fourier.c

```diff
@@ -419,11 +428,11 @@ fourier_create_procedure(GimpPlugIn *plug_in,
 #endif
 
     gimp_procedure_set_menu_label(procedure, _("_Fourier..."));
-    gimp_procedure_add_menu_path(procedure, PLUG_IN_MENU_LOCATION);
+    gimp_procedure_add_menu_path(procedure, _(PLUG_IN_MENU_LOCATION));
```

Comment:
I noticed the menu location dancing around as I tried variations of `<imagem>/filtros/genericos`. Since fourier.c is using the `<image>/filters/generic` location, you are correct, that can be simplified.
On the other hand, if you're adding a folder, and putting several plugins into the same folder, like in [gimp-plugin-astronomy](https://github.com/JoesCat/gimp-plugin-astronomy/blob/master/src/alignment.c#L237), then it makes sense to do this, this way you can translate the last folder Astronomy into the various locals.

Reply:
if we add a folder one day, we need to have the "known part" as untranslatable and the new folder as translatable as mentionned in the documentation
```
PLUG_IN_MENU_LOCATION_KNOWN_PART + _(PLUG_IN_MENU_LOCATION_NEW_PART)
```

For now I think it is best not to translate to avoid confusion for potential other translators.